lib/Kernel/utils.pm

```diff
@@ -0,0 +1,45 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package Kernel::utils;
+
+use base Exporter;
+use Exporter;
+
+use strict;
+use warnings;
+use testapi;
+
+our @EXPORT = qw(
+  get_available_modules
+);
+
+=head1 SYNOPSIS
+
+Kernel testing helpers.
+
+=cut
+
+=head2 get_available_modules
+
+ get_available_modules();
+
+ Temporary helper to be moved to the registration.pm
```

Comment:
+1 for move to lib/registration.pm (can be reusable for others).

Reply:
Why not to move it directly?